_episodes/02-working-with-environments.md

```diff
@@ -146,18 +146,20 @@ $ conda create --name python36-env python=3.6
 > {: .solution} 
 {: .challenge}
 
-You can create a Conda environment and install multiple packages by listing the packages 
-that you wish to install.
+You can create a Conda environment and install multiple packages by listing the packages that you wish to install,
+optionally including the version you wish to use.
 
 ~~~
-$ conda create --name basic-scipy-env ipython=7.13 matplotlib=3.1 numpy=1.18 scipy=1.4
+$ conda create --name basic-scipy-env ipython matplotlib=3.1 numpy=1.24.1 scipy=1.10.0
```

Comment:
Why remove the `ipython` version?

Reply:
The previous example stated that "_optionally including the version you wish to use._" and so I thought it would be useful to show in the example how you can mix and match explicit version and the most recent version (as is the case with `ipython` here).

Reply:
Left as is, but happy to put version back in if you think it should be, just let me know.

Reply:
Fine to leave.